README.md

```diff
@@ -54,9 +54,23 @@ javafx {
 }
 ```
 
+## System Configuration
+
+### Mac OS
+
+To let TestFX manage Mouse and Keyboard, you have to authorized it.
+
+* Go to System Setting > Security and Privacy > Accessibility
+    * If you want to exectute the tests from your IDE, check your IDE in the list
+    * If you want to exectute the tests from maven/gradle in your terminal, check the Terminal
+    * If you don't find the Terminal or your IDE, click on + and add it to the list
```

Comment:
If your terminal or IDE are not listed then click on `+` and add it to the list.